mkdocs.yml

```diff
@@ -23,7 +23,10 @@ nav:
           - Ethereum: topics/blockchain/ethereum.md
           - Features: topics/blockchain/features.md
       - DeFi:
-          - Liquidity provider: topics/defi/liquidity-provider/sand-staking.md
+          - Liquidity provider:
+              - SAND staking: topics/defi/liquidity-provider/sand-staking.md
+      - Catalyst:
+          - Gems, catalyst and GemsCatalystsRegistry: topics/catalyst/catalyst.md
```

Comment:
Title too long, let's go with something simplier maybe:
`Gems, catalyst and GemsCatalystsRegistry` => `Gems & Catalysts`